examples/getting-started-session-based/03-serving-session-based-model-with-Torch-backend.ipynb

```diff
@@ -0,0 +1,1473 @@
+{
```

Comment:
<p>Can we print the train_dict? or at least one key that the user can see, what the structure would be like?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>